pkg/resources/management.cattle.io/v3/roletemplate/RoleTemplate.md

```diff
@@ -0,0 +1,16 @@
+## Validation Checks
+
+Note: all checks are bypassed if the RoleTemplate is being deleted
+
+###  Circular Reference
+
+Circular references to webhooks (a inherits b, b inherits a) are not allowed. More specifically, if "roleTemplate1" is included in the `roleTemplateNames` of "roleTemplate2", then "roleTemplate2" must not be included in the `roleTemplateNames` of "roleTemplate1". This checks prevents the creation of roles whose end-state cannot be resolved.
+
+### Rules Without Verbs 
+
+Rules without verbs are not permitted. The `rules` included in a roleTemplate are of the same type as the rules used by standard kubernetes RBAC types (such as `Roles` from `rbac.authorization.k8s.io/v1`). Because of this, they inherit the same restrictions as these types, including this one.
+
+### Escalation Prevention
+
+Users can only change RoleTemplates with rights less than or equal to those they currently possess. This prevents privilege escalation. 
```

Comment:
```suggestion
Users can only change RoleTemplates with rights less than or equal to those they currently possess. This prevent privilege escalation. 
```

Reply:
Unlike the other places where this is stated, this does not include a `to` before, so the `prevents` reads more naturally than the `prevent` does. Because of this, I will leave this one as-is.